index.js

```diff
@@ -40,8 +40,11 @@ var isInWatchMode = function(){
   var argv = yargs(process.argv)
       .alias('w', 'watch')
       .argv;
+  var hasWebpackDevServer = Array.prototype.filter.call(process.argv, function (arg) {
+    return arg.indexOf('webpack-dev-server') !== -1;
+  }).length > 0;
 
-  return typeof argv.watch !== "undefined" && argv.watch;
+  return hasWebpackDevServer || (typeof argv.watch !== "undefined" && argv.watch);
```

Comment:
🎨  I think more possibly  it might make sense to put `(typeof argv.watch !== "undefined" && argv.watch)` into an arg so that this line is less noisy